packages/common/src/hooks/useRowRenderCount/useRowRenderCount.test.tsx

```diff
@@ -42,20 +43,19 @@ describe('useRowRenderCount', () => {
       paginationRowHeight = 0,
       saveButtonRowHeight = 0,
       footerHeight = 0
-    ) =>
-    ({ children }: { children: ReactNode[] }) => {
+    ): FC =>
+    ({ children }) => {
       return (
         <TestingProvider>
-          <TestingRouter initialEntries={['']}>
-            <ThemeChangerer
-              paginationRowHeight={paginationRowHeight}
-              dataRowHeight={dataRowHeight}
-              headerRowHeight={headerRowHeight}
-              footerHeight={footerHeight}
-              saveButtonRowHeight={saveButtonRowHeight}
-            >
-              {children}
-            </ThemeChangerer>
+          <ThemeChangerer
+            paginationRowHeight={paginationRowHeight}
+            dataRowHeight={dataRowHeight}
+            headerRowHeight={headerRowHeight}
+            footerHeight={footerHeight}
+            saveButtonRowHeight={saveButtonRowHeight}
+          />
+          <TestingRouter initialEntries={['/']}>
+            <Route path="/" element={<>{children}</>} />
```

Comment:
Part of why this was larger than expected was because there were a few tests that were failing, I think, from my previous PRs.

I wanted to make sure everything was passing from this somewhat sweeping change, so I fixed them up in here.

This test was failing because of some change where `initialEntries={['']}` was causing nothing to render. I suspect from my change of the lock file 🙄